components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/PreferenceRetrievalClient.java

```diff
@@ -56,6 +56,7 @@
     private static final String USERNAME_RECOVERY_PROPERTY = "Recovery.Notification.Username.Enable";
     private static final String NOTIFICATION_PASSWORD_RECOVERY_PROPERTY = "Recovery.Notification.Password.Enable";
     private static final String QUESTION_PASSWORD_RECOVERY_PROPERTY = "Recovery.Question.Password.Enable";
+    private static final String SELF_SIGN_UP_LOCK_ON_CREATION_PROP = "SelfRegistration.LockOnCreation";
```

Comment:
```suggestion
    private static final String SELF_SIGN_UP_LOCK_ON_CREATION_PROPERTY = "SelfRegistration.LockOnCreation";
```

Reply:
Shall we rename the variable similar to other properties? WDYT

Reply:
Fixed with ee05c66034e8bece65bfdc14b4c36f6e8e23e6aa